backend/src/db/crud/survey.py

```diff
@@ -132,7 +132,7 @@ def get_all_surveys_user_can_view(
 def get_all_users_with_access_to_survey_count(survey_id: int, session: Session) -> int:
     statement = select(func.count(AccessToViewResults.id)).where(
         (AccessToViewResults.survey_id == survey_id)
-        & (AccessToViewResults.is_deleted == False)
+        & (AccessToViewResults.is_deleted is False)
```

Comment:
I think this caused the code to not work properly

Reply:
Yes, this must be ==